projects/ad40xx_fmc/zed/system_bd.tcl

```diff
@@ -1,5 +1,6 @@
 
 source $ad_hdl_dir/projects/common/zed/zed_system_bd.tcl
+source $ad_hdl_dir/projects/scripts/adi_pd_intel.tcl
```

Comment:
Should this be adi_pd ?

Reply:
> Can you make sure that you're not overloading the log file with unnecessary prints?

yes. i will double check this at the end however i would be inclined to keep some of them since at one point there were some questions regarding the contents and printing information before it is converted to hex would be useful

Reply:
i think is stands for Product Detection. could you suggest a different name?

Reply:
The line is :source $ad_hdl_dir/projects/scripts/adi_pd**_intel**.tcl , but the rest of the additions add source $ad_hdl_dir/projects/scripts/adi_pd.tcl

Reply:
fixed